rules/aip0141/count.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0141
+
+import (
+	"strings"
+
+	pluralize "github.com/gertd/go-pluralize"
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+)
+
+var count = &lint.FieldRule{
+	Name: lint.NewRuleName("core", "0141", "count-suffix"),
+	URI:  "https://aip.dev/141#guidance",
+	LintField: func(f *desc.FieldDescriptor) []lint.Problem {
+		if n := f.GetName(); strings.HasPrefix(n, "num_") {
+			want := pluralize.NewClient().Singular(n[4:]) + "_count"
```

Comment:
Will people use `num_of_books` or `number_of_books` as a quantity field? For `num_of_books`, we will then produce a wired suggestion `of_book_count`.

Reply:
I am not trying to address every possible contingency, and will make a separate rule later prohibiting prepositions.

Reply:
Thanks!